exporter/opencensusexporter/opencensus.go

```diff
@@ -139,7 +139,15 @@ func (oce *ocagentExporter) PushMetricsData(ctx context.Context, md consumerdata
 	err := exporter.ExportMetricsServiceRequest(req)
 	oce.exporters <- exporter
 	if err != nil {
-		return len(md.Metrics), err
+		return numTimeSeries(md), err
 	}
 	return 0, nil
 }
+
+func numTimeSeries(md consumerdata.MetricsData) int {
```

Comment:
Seems to be a copy of the function in metrics_helper.go. Is duplication deliberate?

Reply:
Done.